src/containers/flatVector2D.tpp

```diff
@@ -1091,6 +1091,10 @@ const T* FlatVector2D<T>::operator[](std::size_t i) const
 {
     assert(isIndexValid(i));
 
+    if(m_v.empty()) {
```

Comment:
nit: could you add a space between if and the parenthesis to be consistent with the code-style of this file?